css/css-backgrounds/background-clip/clip-text-multi-line.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<title>CSS Test: background-clip: text paints multi-line text in the wrong place</title>
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds-4/#valdef-background-clip-text">
```

Comment:
Maybe have a second link to https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-color ?